internal/components/components.go

```diff
@@ -208,6 +209,7 @@ func Components() (otelcol.Factories, error) {
 		oidcauthextension.NewFactory(),
 		sigv4authextension.NewFactory(),
 		zpagesextension.NewFactory(),
+		jaegerremotesampling.NewFactory(),
```

Comment:
Nit: These are generally in alphabetical order, so it'd be nice to move this up to be in its proper place. I see a couple out of place already, but it'd still be nice.